Bottom: 0061c21c690bf77edcf8500a7fbdf405db7d2152
Top:    81bf2f2383331d92bed00e35d1d5e58967ee00ae
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-13 15:45:36 -0500

provision production server


---
diff --git a/provision.sh b/provision.sh
index 3762e89d..d58ff0fa 100755
--- a/provision.sh
+++ b/provision.sh
@@ -1,7 +1,7 @@
 #! /bin/bash
 
 #PYTHONUNBUFFERED=1 \
-     
+
 ANSIBLE_ENABLE_TASK_DEBUGGER=True \
 ANSIBLE_FORCE_COLOR=true \
 ANSIBLE_HOST_KEY_CHECKING=false \
@@ -9,6 +9,6 @@ ANSIBLE_SSH_ARGS='-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o Contr
 exec ansible-playbook \
         --connection=ssh \
         --timeout=30 \
-        --inventory-file=ansible-playbook/staging \
+        --inventory-file=ansible-playbook/production \
         -vv \
         ansible-playbook/site.yml "$@"
